demo/Demo.csproj

```diff
@@ -2,16 +2,17 @@
     <PropertyGroup>
         <OutputType>WinExe</OutputType>
         <TargetFramework>net7.0</TargetFramework>
+		<AvaloniaNameGeneratorIsEnabled>false</AvaloniaNameGeneratorIsEnabled>
```

Comment:
Could you please reduce the indention to match the rest of the file?

Reply:
Nice find, my bad!
All done 🙂